src/library/data.js

```diff
@@ -1,22 +1,39 @@
+import bodhi from "../images/bodhi.png";
+import mamalicious from "../images/mamalicious.png";
+import azeitona from "../images/azeitona.png";
+import vincentvegan from "../images/vincentvegan.png";
+import rawlikesushi from "../images/rawlikesushi.png";
+import biolose from "../images/biolose.png";
+import stueckgut from "../images/stueckgut.png";
+import glore from "../images/glore.png";
+import vunderland from "../images/vunderland.png";
+import wochenmarkt from "../images/wochenmarkt-eppendorf.png";
+import schnittstubeverde from "../images/schnittstube-verde.png";
+import stinevega from "../images/stine-vega.png";
+import heikevossberg from "../images/naturkosmetikum-heikevossberg.png";
+import mimulus from "../images/mimulus.png";
+import jeannie from "../images/jeannie.png";
+
```

Comment:
How about importing images from outside as URLs? In the long run you'd have URLs anyway (through API) - but that's just a suggestion ;)

Reply:
That's a good point, but as long as don't use an API, I prefer to use downloaded images instead of URLs :)